index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Birds?</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Karla:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./normalize.css">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <main>
+    <body>
+      <header>
+        <section class="nav-block">
+          <div class="site-name">
+            <img src="./icons/burb.png" class="logo">
+            <h1 class="headline">irds?</h1>
+          </div>
+          <div class="navigation">
+            <div class="nav-group">
+              <img src="./icons/icons8-share-50.png" class="icon">
+              <h2 class="share">Spread The Word</h2>
+            </div>
+            <div class="nav-group">
+              <img src="./icons/icons8-bulleted-list-50.png" class="icon">
+              <h2 class="bird-finder">"Birds"</h2>
+            </div>
+            <div class="nav-group">
+              <img src="./icons/icons8-customer-50.png" class="icon">
+              <h2 class="affiliates">Affiliates</h2>
+            </div>
+          </div>
+        </section>
+        <section class="user-name">
+          <img src="./icons/icons8-cat-profile-50.png" class="icon">
+          <h2 class="user">Admin Username &#9660;</h2>
+        </section>
+      </header>
+        <section class="grid-title">
+          <h2>Recent birdTalks</h2>
+        </section>
+        <div class="grid-box">
```

Comment:
Have you considered that the `div` of ``grid-box` should be the section and then subsequent `info-box` should instead be an article? It'll make your site more tab-able (sp?).